test/run_tests.py

```diff
@@ -1,35 +1,43 @@
 #!/usr/bin/env python
-
 """Script to run Selenium2Library acceptance tests.
 
 Tests are executed using Robot Framework and results verified automatically
 afterwards using `robotstatuschecker` tool. The tool can be installed using
 `pip install robotstatuschecker` and more information about it can be found
-from https://github.com/robotframework/statuschecker/. Notice that initially
-some tests fail.
-
-Usage:
+from: https://github.com/robotframework/statuschecker/. Notice that initially
+some tests may fail.
 
-  run_tests.py interpreter browser [options]
+When running test by using browser from Sauce labs, it is required that the
+Sauce Connect is used. The Sauce Connect allows the browser from Sauce Labs
+reach the acceptance test web server. The acceptance test uses tunnel with
+name `localtunnel` and therefore when establishing the Sauce Connect tunnel
+use the following command:
+    `bin/sc -u YOUR_USERNAME -k YOUR_ACCESS_KEY -i localtunnel`
 
-Arguments:
+More details and to downlaod Sauce Connect visit:
+https://wiki.saucelabs.com/display/DOCS/High+Availability+Sauce+Connect+Setup
 
-  interpreter:  Any Python interpreter supported by the library (e.g. `python`,
-                `jython`, `c:\\Python27\\python.exe`)
-  browser:      Any browser supported by the library (e.g. `chrome`, `firefox`)
-  options:      Additional command line options passed to Robot Framework
+It is possible to pass Robot Framework command line arguments to the test
+execution as a last arguments to the `run_tests.py` command. It is
```

Comment:
Same correction as [in comment](https://github.com/robotframework/Selenium2Library/pull/716/files#r95268607)